swatch-metrics/README.md

```diff
@@ -0,0 +1,90 @@
+# swatch-metrics
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+## Running the application in dev mode
+
+You can run your application in dev mode that enables live coding using:
+
+```shell script
+./gradlew :swatch-metrics:quarkusDev
```

Comment:
Running `./gradlew :swatch-metrics:quarkusDev` doesn't work right now due to this:

```
Caused by: io.smallrye.config.ConfigValidationException: Configuration validation failed:
        java.util.NoSuchElementException: SRCFG00014: The config property rhsm-subscriptions.metering.prometheus.metric.jobMaxAttempts is required but it could not be found in any config source
        java.util.NoSuchElementException: SRCFG00014: The config property rhsm-subscriptions.metering.prometheus.metric.jobBackOffInitialInterval is required but it could not be found in any config source
        java.util.NoSuchElementException: SRCFG00014: The config property rhsm-subscriptions.metering.prometheus.metric.jobBackOffMaxInterval is required but it could not be found in any config source
        java.util.NoSuchElementException: SRCFG00014: The config property rhsm-subscriptions.metering.prometheus.metric.eventSource is required but it could not be found in any config source
```

The properties being referenced are set in either the `metrics-rhel` or `metrics-telemeter` (oopenshift-metering-worker) profiles currently.

I tried doing `QUARKUS_PROFILE=openshift-metering-worker ./gradlew :swatch-metrics:quarkusDev`, but then there are other exceptions (splunk hec http calls failing because the dev profile usually disables it) happening since the dev profile isn't active.

It'd be ideal if we can get the default behavior of `./gradlew :swatch-metrics:quarkusDev` to use property values from both `metrics-telemeter`, `dev` - with precedence given to `metrics-telemeter`.

More generally, I'd like to see us move away from using profiles to separate swatch-metrics and swatch-metris-rhel configs, and only have environment-related profiles (eg `test`,`dev`,`ephemeral`,`stage`,`prod`).  Not something that needs to be done as part of this card though.

Reply:
I also think it's more natural to only have tne environmenta-related profiles. And also it makes easy to use the service when running either on DEV or on test modes. 
I've updated the pull request to get rid of the metrics-telemeter and metrics-rhel profiles.